client/app/assets/styles/variables.js

```diff
@@ -139,8 +139,10 @@ module.exports = {
 
   '--MenuItem_borderColor': topbarBorderColor,
   '--MenuItem_backgroundColorHover': backgroundLightColorHover,
-  '--MenuItem_paddingTopbarVertical': paddingTopbarVertical,
-  '--MenuItem_paddingTopbarHorizontal': paddingTopbarHorizontal,
+  '--MenuItem_paddingTopbarVertical': '0.92857em',
+  '--MenuItem_paddingTopbarHorizontal': '1.7142857em',
+  '--MenuItem_paddingOffScreenVertical': '0.58823529411em',
+  '--MenuItem_paddingOffScreenHorizontal': '1.41176470588em',
```

Comment:
The em value reminds me of this: https://twitter.com/pcwalton/status/718565939432718336

Where do you get these values and do we really need that kind of precision?